src/frontend/scss/_colors.scss

```diff
@@ -5,6 +5,7 @@
 $black: #000000;
 $white: #ffffff;
 $gray7: #111111;
+$gray17: #2b2b2b;
```

Comment:
Why not reuse gray20 instead of making a new arbitrary gray17?

Reply:
gray20 is to dark compared to gray17, at the beginning it was gray20 but i needed to better fit to the funmooc homepage which use gray17.

Reply:
Ok. I was just thinking that 7 different grays is starting to feel like a lot and 17-20 is not too far (if 0 is black and 100 is white)

Reply:
Designers often use a lot of same color tones to make a design, this can be justified for backgrounds which cover a lot more surface than font and so very significant for our eye.